packages/cjam/0.6.5/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# CJam depends on Java
+source ../../java/15.0.2/build.sh
+
+curl -L "https://downloads.sourceforge.net/project/cjam/cjam-0.6.5/cjam-0.6.5.jar?ts=gAAAAABgf6xp4bmSJmNMjz8z23zWVbRgW6mhaFrnxdBe_riqC4rZnhUpXJrQzPpSqhniaoGptfkI_r-tjjBc5326oRmloXIJYQ%3D%3D&r=https%3A%2F%2Fsourceforge.net%2Fprojects%2Fcjam%2Ffiles%2Fcjam-0.6.5%2Fcjam-0.6.5.jar%2Fdownload" -o bin/cjam.jar
+
+# Really executable jar
+cat <<EOF > bin/cjam
+#!/bin/sh
+exec java -jar \$0 "\$@"
+
+
+
+EOF
```

Comment:
Any specific reason for this, can this not just be done in `run`?

Reply:
no reason other than just being a cool way to run a jar as a 'standalone' executable and I didn't want to create another env var to reference the jar path. I saw that dragon lang was technically doing this, but it's pre-built like that. I'll change it to be the normal way then for cjam.